src/components/Login/RegisterForm.js

```diff
@@ -150,6 +150,7 @@ const RegisterForm = ({ dispatch, modal, itemDetail }, props) => {
           itemDetail.ownerId
         )
       );
+      window.location.reload();
```

Comment:
Not sure if anyone else has it but when my homepage reloads, the category images first load horizontally squished then adjust to the proper size. Because of this, the page reload after login from the homepage looks weird. 

Also, in the current workflow, if a user is not logged in but tries to make an offer, they are prompted with the login form. Once logged it, it immediately opens the make offer form. With the page reload, the offer form displays for a quick second, then the page reloads to just the item details page.

I'm not sure if the page reload is the right addition

Reply:
oh true. I think i noticed sometimes with the google login the page doesnt actually change when logging in (very rarely occurs but yeah no need to merge)